deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/util/ValidationUtils.java

```diff
@@ -0,0 +1,318 @@
+package org.deeplearning4j.util;
+
+import java.util.Arrays;
+import org.nd4j.base.Preconditions;
+
+public class ValidationUtils {
```

Comment:
Brief class javadoc would be good.
Also feel free to add an ```@author``` tag if you want (totally optional).
Also need copyright header at the start of the file (you can copy/paste from elsewhere)